### ISSUE TITLE
Switch to Papyrus perspective by default

### ISSUE THREAD

**Issue by planger** [eclipsesource/papyrus-umllight#25]
Currently we open the resource perspective on a fresh workspace. I think it'd be better to use the Papyrus perspective by default.

**Comment by cdamus** [comment]
I can confirm that this was already implemented this in commit cb6da40f8b7f948f613fee5c8120a049a3c3d15e

```
# Property "org.eclipse.ui/defaultPerspectiveId" controls the 
# perspective that the workbench opens initially
org.eclipse.ui/defaultPerspectiveId=org.eclipse.papyrus.infra.core.perspective
```

and it works for me on new workspaces.

**Comment by cdamus** [comment]
Actually, in testing a product build for #16 I am finding that the product doesn't open the Papyrus perspective by default as happens in a run-time workbench.
